src/validations/chatMessage.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export */
+import Joi from 'joi';
+
+const chatMessageSchema = Joi.object().keys({
+  message: Joi.string().min(1).required().label('Chat message')
+});
```

Comment:
You should provide valid message which shows the erro and include also the message limit which you set to 500 in database

Reply:
done